modules/tensor_mechanics/test/tests/shell/dynamics/tests

```diff
@@ -1,32 +1,43 @@
 [Tests]
-  design = "C0TimoshenkoBeam.md"
-  issues = "#15067"
   [verification_tests]
-    requirement = "The mechanics system shall accurately predict the"
-                  " dynamic behavior of a shell element with clamped-"
-                  "free-free-free conditions when "
-    [./shell_dynamics_bending_moment_free]
+    design = "C0TimoshenkoBeam.md"
```

Comment:
This should be `ShellElement.md`

Reply:
It was wrong to start with.

Reply:
oh, copy paste from the beam element verification?

Reply:
Probably